tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -172,3 +179,37 @@ Check GPU Resources
         Page Should Not Contain Element     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
     END
 
+Verify Quick Starts Work As Expected When All the Steps Marked As Yes
+    [Arguments]     ${element}
+    Open Element By Name    ${element}
+    ${count}=   Get The Count Of Description
+    FOR     ${index}    IN RANGE    ${count}
+        Move To Next Description By Clicking Yes
+    END
+    Verify Previous Description
+    Move To Next Description By Clicking Yes
+    Verify The Progress Of The Items    ${element}  Complete
+
+Verify Quick Starts Work As Expected When Restarting The Previous One
+    [Arguments]     ${element}
+    Element Text Should Be  //article[@id="${element}"]//a      Restart
+    Open Element By Name    ${element}
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+    Page Should Contain Element     //div[@data-testid="qs-drawer-close"]/button
+    Click Button        //div[@data-testid="qs-drawer-close"]/button
```

Comment:
You could create a dedicated kw to perform this, since the same lines are deplicated later

Reply:
Done !